google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/FlexibleFacetEditor.java

```diff
@@ -294,8 +303,21 @@ public void hyperlinkUpdate(HyperlinkEvent e) {
         Path destinationFilePath = destinationFolderPath.resolve(APP_YAML_FILE_NAME);
 
         if (validateAndWarnFileGeneration(destinationFolderPath, destinationFilePath)) {
-          APP_ENGINE_PROJECT_SERVICE
+          UsageTrackerProvider.getInstance()
```

Comment:
same, not the right place (in fact, no tracking should exist at all in this class)